### ISSUE TITLE
unknown network error

### ISSUE THREAD

**Issue by kasra3422** [archon-runtime/archon-runtime.github.io#24]
Hi,

Android APKs does not have internet access when running over chrome,
chrome has internet access and works, also I checked DNS settings.

Thank you.

![ss_2017_04_11_22 02 43_001_selection](https://cloud.githubusercontent.com/assets/27306703/24928817/10d2615a-1f04-11e7-9a6b-721cbba6e91c.png)
![ss_2017_04_11_22 09 55_001_selection](https://cloud.githubusercontent.com/assets/27306703/24928844/268d3222-1f04-11e7-9211-63840d57f15c.png)
![ss_2017_04_11_22 09 22_001_selection](https://cloud.githubusercontent.com/assets/27306703/24928848/2a396fe4-1f04-11e7-89fe-46fad4cb17db.png)

**Comment by Heybye44** [comment]
I'm having the same issue on the same version of ARChron which is the latest on the website. Although, I've only tried Instagram so far. Are you on Ubuntu 16.04 also?

**Comment by kasra3422** [comment]
Yes, I'm using Ubuntu 16.04. I think I should test it on windows.

**Comment by ghost** [comment]
another ubuntu user here confirming this issue, on ubuntu 17.10. chrome has working internet access but kik will not login.
